scripts/coverage_helper.py

```
@@ -0,0 +1,116 @@
+        TEMP.mkdir(parents = True, exist_ok = True)
+        removed_patterns = 0
+        for pattern in ("coverage-*.xml", ".coverage-*.sqlite"):
+            for coverage_file in TEMP.glob(pattern):
+                coverage_file.unlink()
+                removed_patterns += 1
+        removed_paths = 0
+        for path in (TEMP / "coverage.sqlite", ROOT / ".coverage"):
+            if path.exists():
+                path.unlink()
+                removed_paths += 1
+    except Exception as exc:  # noqa: S110 suppress to log
```

Ruff (0.14.4)

Unused `noqa` directive (unused: `S110`)

Remove unused `noqa` directive

(RUF100)